lessons/Projects/project-research-paper/langs/en-us/pages/research-project-rubric-2.adoc

```diff
@@ -0,0 +1,38 @@
+= Rubric: Research Project (2)
```

Comment:
I'm not clear on how this should be filled out. Is the idea that students fill in the first column ONLY, and then just put a checkbox in one of the other cells in each row? If so, surely there's a more efficient use of space here?

Reply:
maybe we can chat about this too at our one-on-one.

I thought teachers might like to leave specifc notes / feedback for students. But I suppose that's not obvious. I could either offer specific teacher directions re: how to use the rubric, or I could add way more lines and assume that teachers will just put a checkmark. Or maybe I could just provide ONE longer cell for teacher feedback, rather than 3 little ones?